cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/cim14/Cim14SmallCasesNetworkCatalog.java

```diff
@@ -175,40 +177,27 @@ public Network smallcase1() {
     }
 
     public Network ieee14() {
-        return cimImport(catalog.ieee14());
+        return loadNetwork(catalog.ieee14());
     }
 
     public Network nordic32() {
-        return cimImport(catalog.nordic32());
+        return loadNetwork(catalog.nordic32());
     }
 
     public Network m7buses() {
-        return cimImport(catalog.m7buses());
+        return loadNetwork(catalog.m7buses());
     }
 
     public Network txMicroBEAdapted() {
-        return cimImport(catalog.txMicroBEAdapted());
-    }
-
-    private Network cimImport(TestGridModel gm) {
-        try {
-            return cimImport1(gm);
-        } catch (IOException e) {
-            throw new PowsyblException("failed to import CIM1 model " + gm.name(), e);
-        }
+        return loadNetwork(catalog.txMicroBEAdapted());
     }
 
-    private Network cimImport1(TestGridModel gm) throws IOException {
-        ReadOnlyDataSource ds = gm.dataSource();
-
-        Set<String> names1 = ds.listNames(".*");
-        if (LOG.isInfoEnabled()) {
-            LOG.info("List of names in data source for CIM1Importer = {}", Arrays.toString(names1.toArray()));
-        }
-        return new CIM1Importer(Mockito.mock(PlatformConfig.class)).importData(ds, null);
+    private Network loadNetwork(TestGridModel gm) {
+        XMLImporter xmli = new XMLImporter();
```

Comment:
In order to prevent errors due to local configuration, do:
```XMLImporter xmli = new XMLImporter(Mockito.mock(PlatformConfig.class));```

Reply:
added